core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceResponseConverterTest.java

```diff
@@ -557,6 +569,10 @@ private static void shouldBeConvertedByDefaultResponseConverter(WebClient client
         res = aggregated(client.get("/jsonNode"));
         assertThat(res.contentType()).isEqualTo(MediaType.JSON_UTF_8);
         assertThat(res.content().array()).isEqualTo(mapper.writeValueAsBytes(JSONNODE));
+
+        res = aggregated(client.get("/httpFile"));
+        assertThat(res.contentType()).isNull();
```

Comment:
Don't we need to set `application/octet-stream` as a default `content-type` if a user does not specify a `content-type`?

Reply:
Thanks for the catch.
`AbstractHttpFile#addCommonHeaders` is the right place for the change.

I will make an additional commit for this.

Reply:
Sorry, after reading `HttpFile` source code. No `content-type` seems to be intended only for an `HttpData` based HttpFile`.
Please ignore this review. 😅